web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputer/Setup.tsx

```diff
@@ -117,8 +151,37 @@ function Information(props: { onSetUpAgentClick(): void }) {
   );
 }
 
-function AgentSetup({ rootClusterUri }: { rootClusterUri: RootClusterUri }) {
+function AccessError(props: { access: ConnectMyComputerAccessNoAccess }) {
+  switch (props.access.reason) {
+    case 'unsupported-platform': {
+      return (
+        <Alert mb={0}>
+          Connect My Computer is not supported on your operating system.
+        </Alert>
+      );
+    }
+    case 'insufficient-permissions': {
+      return (
+        <Alert mb={0}>
+          You have insufficient permissions to use Connect My Computer.
```

Comment:
should these have more actionable messages? for example:

```
              You don’t have sufficient permissions to create an integration.
              Reach out to your Teleport administrator to request additional
              permissions.
```

Reply:
Or at least we could leave a link to docs.

Reply:
Thanks, I made this particular message more actionable and I added a link to the docs to each message.

I didn't think of it originally because those users are not supposed to see the deep link in the first place – I forgot to mention that in the PR description. I think it'll be rather rare for an actual user to end up in this tab with insufficient permission. But yeah, it wasn't much work to make those messages better.